modules/services/gpg-agent.nix

```diff
@@ -203,10 +203,11 @@ in
         [ cfg.extraConfig ]
       );
 
-      home.sessionVariables =
-        optionalAttrs cfg.enableSshSupport {
-          SSH_AUTH_SOCK = "$(${gpgPkg}/bin/gpgconf --list-dirs agent-ssh-socket)";
-        };
+      home.sessionVariablesExtra = optionalString cfg.enableSshSupport ''
+        if [[ -Z "$SSH_AUTH_SOCK" ]]; then
```

Comment:
```suggestion
        if [[ -z $SSH_AUTH_SOCK ]]; then
```

Reply:
Thanks, fixed `-z` but kept quotes if you don't mind.

Reply:
Right, they're entirely optional there (because of `[[`) but they don't hurt either.